src/Couchbase.Lite.Shared/Database.cs

```
@@ -1769,7 +1774,12 @@ internal bool ProcessAttachmentsForRevision(RevisionInternal rev, IList<Revision
         internal IDictionary<string, object> GetAttachmentsFromDoc(string docId, RevisionID revId)
         {
             var rev = new RevisionInternal(docId, revId, false);
-            LoadRevisionBody(rev);
+            try {
+                LoadRevisionBody(rev);
+            } catch(Exception) {
+                return null;
```

Ignoring exceptions can be a code smell … what's the reason for doing so here?